### ISSUE TITLE
If a tab is open long enough, most keybindings stop working

### ISSUE THREAD

**Issue by macintacos** [brookhong/Surfingkeys#1300]
## Prelude
* Could you find answer in below documents?

    1. [FAQ](https://github.com/brookhong/Surfingkeys/wiki/FAQ) / [常见问题](https://github.com/brookhong/Surfingkeys/wiki/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98) - NO
    1. [README](https://github.com/brookhong/Surfingkeys/blob/master/README.md) / [读我](https://github.com/brookhong/Surfingkeys/blob/master/README_CN.md) - NO

* Have you searched your problem in [issues](https://github.com/brookhong/Surfingkeys/issues)? - YES, didn't find anything useful

## Error details

SurfingKeys: 0.9.65

Browser: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4147.89 Safari/537.36

URL: Seems to happen on any page that is open long enough

## Context

I don't quite understand how it happens, but it seems that if I open a tab and leave it alone for long enough, when I return to that tab almost no keybindings work anymore. I say _almost_ no keybindings,because things like `t` and `T` don't work, but `j`/`k` for scrolling do. It seems like anything that requires bits of UI to load just don't work.

Is there a timeout of some sort that causes this issue to occur? Once I refresh the page all the keybindings work as expected until they don't (if I leave the page alone for long enough).

**Comment by brookhong** [comment]
Is this still a problem for you? Is there any popup from Surfingkeys when it happens?

**Comment by laphylee** [comment]
@brookhong Yes, it still happen to me although I updated it to the latest version 1.0.5. There is no popup from Surfingkeys. Look forward to the fix. Thanks!

**Comment by b0o** [comment]
Can you share your configuration? Are there any messages in the JavaScript console?

**Comment by laphylee** [comment]
@b0o This is the error message from devtools console. It indeeds come from the Surfingkeys after I confirmed the extention id.

![image](https://user-images.githubusercontent.com/13502835/156952721-c81fb369-a588-4940-b197-fbff206506cf.png)

**Comment by daveleeyee2022** [comment]
It happen to me while I use old version (maybe 1.0? I dont remember) + "const { }=api" + "mapkey" , and when I update to 1.05 and delete "const { }=api"，change all "mapkey“ with “api.mapkey“，it never happen again .

**Comment by ariel-frischer** [comment]
I'm getting this on 1.1.0 macOS Monterey
